server/src/main/java/org/apache/druid/metadata/SQLMetadataStorageActionHandler.java

```diff
@@ -82,7 +82,9 @@ public SQLMetadataStorageActionHandler(
   )
   {
     this.connector = connector;
-    this.jsonMapper = jsonMapper;
+    //noinspection UnnecessaryFullyQualifiedName
+    this.jsonMapper = jsonMapper.copy().addMixIn(org.apache.druid.metadata.PasswordProvider.class,
```

Comment:
Oh, I missed this part. Would you tell me why we should use fully qualified name here?

Reply:
PasswordProvider is in the core module, but shares the same package name. This introduced some issues importing it into the server module. If I use the import statement at the top of the file I get hit with an unused import issue, and If I don't import, the IDE will figure out how to resolve but at runtime you get a class not found error. I decided on the fully qualified name with the noinspection flag. At the same time I'm not a Java veteran so I don't know what the standard would be here

Reply:
I see. I think it's fine, but would you please add a comment about why we use the fully qualified name here?

Reply:
Pushed a comment just now